lua/pl/compat.lua

```diff
@@ -225,9 +225,9 @@ end
 -- This functions mimics the `warn` function added in Lua 5.4.
 -- @function warn
 -- @param ... any arguments
-if not warn then  -- luacheck: ignore
+if not rawget(_G, warn) then
```

Comment:
Sorry I missed the release prep party yesterday, but...

Doesn't this need to be `rawget(_G, "warn")` with the quotes around `warn`)?

Reply:
Followup [PR here](https://github.com/lunarmodules/Penlight/pull/439)

Reply:
oops